css/src/assets/styles/sass/style.scss

```diff
@@ -0,0 +1,230 @@
+@import "reset";
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+body {
+  font-family: "museo-sans",sans-serif;
+  background: #f5f5f5;
+  font-size: 16px;
+  padding: 0.5em;
+}
+img {
+  max-width: 100%;
+}
+a {
+  text-decoration: underline;
+}
+.tile {
+  position: relative;
+  -moz-box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  -webkit-box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  box-shadow: 0 4px 3px 0 rgba(190,190,190,0.5);
+  width: 400px;
+  max-width: 100%;
+  margin: 1em auto 0;
+  border-radius: 8px;
+  background: white;
+  cursor: pointer;
+
+  #tile__love {
```

Comment:
I'd recommend avoiding applying of styles to `id`'s. Using `id`'s for styles can cause specificity headaches down the road when working on a large application.